### PR TITLE
PREAPPS-5046: Send ImportDataRequest with manual refresh request for external accounts

### PR DESCRIPTION
**Pull request by sachinpatel88**

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -956,6 +956,11 @@ export class ZimbraBatchClient {
 			);
 		});
 
+	public getImportExternalAccountStatus = () =>
+		this.jsonRequest({
+			name: 'GetImportStatus'
+		}).then((result: any) => result);
```
Comment (silentsakky) [inline src/batch-client/index.ts:962]: you don't need then

File: src/batch-client/index.ts [hunk 2]
```diff
@@ -956,6 +956,11 @@ export class ZimbraBatchClient {
 			);
 		});
 
+	public getImportExternalAccountStatus = () =>
+		this.jsonRequest({
+			name: 'GetImportStatus'
```
Comment (silentsakky) [inline src/batch-client/index.ts:961]: usually most of the function and schema names are aligned with api name, to reduce confusion

File: src/batch-client/index.ts [hunk 3]
```diff
@@ -956,6 +956,11 @@ export class ZimbraBatchClient {
 			);
 		});
 
+	public getImportExternalAccountStatus = () =>
```
Comment (silentsakky) [inline src/batch-client/index.ts:959]: getImportStatus

File: src/schema/schema.graphql [hunk 4]
```diff
@@ -853,6 +853,18 @@ input ExternalAccount {
...
 }
 
+type ExternalAccountStatus {
+	id: ID
+	isRunning: Boolean
+	success: Boolean
+	error: String
+}
+
+type ExternalAccountStatusResponse {
```
Comment (silentsakky) [inline src/schema/schema.graphql:863]: ImportStatusResponse

File: src/schema/schema.graphql [hunk 5]
```diff
@@ -853,6 +853,18 @@ input ExternalAccount {
 	password: String!
 }
 
+type ExternalAccountStatus {
```
Comment (silentsakky) [inline src/schema/schema.graphql:856]: ImportStatus

File: src/batch-client/index.ts (hunk 2 above)
Comment (sachinpatel88) [reply src/batch-client/index.ts:961]: Yeah, Right. I was trying to make method name self-explanatory. 😄

**Pull request merged**